### PR TITLE
Update to RC2

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

 

 ### Linked Issues ###
 

 - Fixes #

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

**Review by pictos — APPROVED** [review]
:shipit:

File: samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
-    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
```
Comment (BurkusCat) [inline samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj:5]: @VladislavAntonyuk do you know why `and '$(MSBuildRuntimeType)' == 'Full'` was removed from this file?

Comment (VladislavAntonyuk) [reply samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj:5]: Because it was removed from MAUI template

Comment (BurkusCat) [reply samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj:5]: Thank you, I can see the reason why here which sounds like a problem I'm getting on Windows:
https://github.com/dotnet/maui/pull/5875
https://github.com/dotnet/maui/issues/5886

**Pull request merged**